src/components/Table/Table.js

```diff
@@ -0,0 +1,136 @@
+import _ from "lodash";
+import { MDCRipple } from "@material/ripple";
+
+import tableTemplate from "./Table.html";
+import { DotsMenuComponent } from "../DotsMenu/DotsMenu";
+
+const ORDER_TYPES = {
+  0: "INITIAL",
+  1: "ASC",
+  2: "DESC"
+};
+
```

Comment:
const INITAL_ORDER = 0
const ASC_ORDER = 1
const DESC_ORDER = 2

const ORDER_TYPES = [INITAL_ORDER, ASC_ORDER, DESC_ORDER]

Reply:
why do you need this text representation? numeric constants should be fine

Reply:
if (ORDER_TYPES[currentOrderTypeIndex] === "INITIAL") {
      this.state.data = initialData;
    } else {
      this.state.data = _.orderBy(
        data,
        this.state.columnName,
        ORDER_TYPES[currentOrderTypeIndex].toLowerCase()
      );
    }
It is required for the logic to be worked. It contains order type which I pass in to the orderBy function as an argument.

Reply:
const INITIAL_ORDER = 0

...

if (currentOrderTypeIndex === INITIAL_ORDER )

Reply:
oh, I see, you need to pass smth to _.orderBy

Reply:
Yeah

Reply:
then, 

const INITIAL_ORDER = 0
const ASC_ORDER = 1
const DESC_ORDER = 2

const ORDER_TYPES = {
  [INITIAL_ORDER]: 'initial',
  [ASC_ORDER]: 'asc',
  [DESC_ORDER]: 'desc'
}

OR

you can add one more `if` and write 'asc'/'desc' directly inside _.orderBy